content/en/platform/corda/5.0-alpha/deploying/prerequisites.md

```diff
@@ -8,24 +8,40 @@ menu:
     weight: 1000
 section_menu: corda-5-alpha
 ---
-<!--https://r3-cev.atlassian.net/browse/DOC-4188-->
-This section lists the third-party prerequisites for multi-worker cluster deployments. <!--For information about the prerequisites for local deployment with the [CorDapp Standard Development Environment (CSDE)](../cordapp-standard-development-environment/csde.html), see [Third-Party Prerequisites for the CSDE](../getting-started/prerequisites.html).-->
 
-## Software Prerequisites
+This section lists the third-party software prerequisites for multi-worker cluster deployments. <!--For information about the prerequisites for local deployment with the [CorDapp Standard Development Environment (CSDE)](../cordapp-standard-development-environment/csde.html), see [Third-Party Prerequisites for the CSDE](../getting-started/prerequisites.html).-->
 
-Corda 5 cluster deployments have been tested with the following:
+### Container Orchestration
 
-| Software      | Version |
-| ----------- | ----------- |
-| Kubernetes (incl. kubectl)      |     |
-| Helm   |  |
+| Software                                | Version |
+|-----------------------------------------|---------|
+| Kubernetes                              | 1.23    |
+| Amazon Elastic Kubernetes Service (EKS) | 1.23    |
+| Helm | 3.9.4 or newer    |
```

Comment:
@nadinequinr3 shouldn't we align the columns ( | ) with the above rows ?

Reply:
> @nadinequinr3 shouldn't we align the columns ( | ) with the above rows ?

That last one was added through GitHub so the formatting was lost but Hugo will render them all the same. I'll update anyway to improve readability.